integration-tests/features/environment.py

```diff
@@ -1,4 +1,11 @@
 import contextlib
+import subprocess
+
+def before_all(context):
+    # Some steps require sudo, so for convenience in interactive use,
+    # we ensure we prompt for elevated permissions immediately,
+    # rather than potentially halting midway through a test
+    subprocess.check_output(["sudo", "echo", "Elevated permissions needed"])
```

Comment:
Wanted to suggest to `assert os.getuid() == 0`, but that wouldn't actually check the presence of the `sudo` binary.
Might be worthwhile to "centralize" the usage of sudo, and skip it completely if we're already uid == 0.